src/main/java/com/transferwise/openbanking/client/api/payment/common/AuthorizationContext.java

```diff
@@ -0,0 +1,22 @@
+package com.transferwise.openbanking.client.api.payment.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class AuthorizationContext {
```

Comment:
I decided to leave it entirely up to the caller of the library to specify the redirect URL to use when exchanging an authorization code for an access token, rather than say creating some kind of interface to determine the URL to use, as I think this decision logic will be quite organisation specific. For example, some might base on database records, some on HTTP request headers, some only defining one URL.